.gitignore

```diff
@@ -1 +0,0 @@
-.DS_Store
```

Comment:
delete this file from your PR

Reply:
done

Reply:
No)

Reply:
Just add this file to your branch and make commit)